docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -136,6 +136,26 @@ The great-circle distance between two lat-long points, both specified
 in decimal degrees. An optional final parameter specifies `KM`
 (the default) or `miles`.
 
+### `GREATEST`
+
+Since: 0.18.0
+
+```sql
+GREATEST(col1, col2...)
+```
+
+The largest value among the provided elements.
```

Comment:
Non-null elements only, right? (Same for LEAST below.)

Reply:
Do you think appending  `returns null if all elements are null, otherwise ignores null values.` would be a clarifying but concise way of communicating this?

Reply:
See https://github.com/confluentinc/ksql/pull/7683#discussion_r661040743